src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final User bob = new User("bob12345@gmail.com", "6h6%m3dS2jAy", 19);
+    private static final User alice = new User("alice23145@gmail.com", "6g!RO008ry^8", 20);
+    private static final User mike = new User("mike2352@gmail.com", "h760BVS1wh!n", 18);

```

Comment:
If you use this object only for one test, don't need to create it as class fields